README.md

```diff
@@ -1,31 +1,198 @@
+
+Default Server Port: 9000</br>
+Default Payload Size: 0 - 1200 bytes</br>
+Default Beaconing Interval: 30 - 60 seconds
+
+### Configurations
+#### Configuring the Client Files
+1. On the client device, open `tcp-client.py` or `udp-client.py` in a text editor such as nano.
+1. Navigate to the line starting with `server =` followed by a list of IP addresses and replace them with the IP Address(es) of your server(s).
+1. (optional) Change the server port at the line with `SERVER_PORT = 9000`. (If you change the port number, you *must* edit the corresponding `-server.py` file to match.)
+1. (optional) Change the payload size range at the line with `m = randint(0, 1200)`.
+1. (optional) Change the beaconing interval at the line with `rsleep = random.randint(30, 60)`.
+1. Save the changes and exit.
+
+#### Configuring Server Files (optional)
+NOTE: This is only necessary if you changed the server port number in a `-client.py` script. For example, if you changed the server port in `tcp-client.py`, you *must* edit `tcp-server.py` script to match.
+1. On the server device, open the corresponding `-server.py` file in a text editor such as nano.
+1. Change the server port number at the line with `SERVER_PORT = 9000` to match the number you chose in the corresponding `-client.py` file.
+1. Save the changes and exit.
+
+### Running the Script Pairs
+#### TCP Beacon Pair
+On the client machine, navigate into the `python-scripts` directory and run:
+```
+python3 ./tcp-client.py
+```
+On the server machine, navigate into the `python-scripts` directory and run:
+```
+python3 ./tcp-server.py
+```
+
+#### UDP Beacon Pair
+On the client machine, navigate into the `python-scripts` directory and run:
+```
+python3 ./udp-client.py
+```
+On the server machine, navigate into the `python-scripts` directory and run:
+```
+python3 ./udp-server.py
+```
+
+</br>
+</br>
+
+## Running Tools in the Background with `screen`
+You can use the `screen` utility to run these scripts in a separate session in the background and access it later. To do so, simply add `screen -S <name> -d -m` at the beginning of the command, replacing `<name>` with a name for the session.
+
+#### Start session in the background:
+```
+screen -S my-session -d -m ./simple-beacon.sh 192.168.56.104
+```
+This will run `simple-beacon.sh` in the background.
+
+If you wish to access the session later and turn it off, you can do so by using `screen -r <name>` to re-attach the session to the terminal window then pressing "Ctrl + C".
+
+#### Re-attaching the session:
+```
+screen -r my-session
+```
+This will re-attach `my-session` to the terminal window. I can then stop `simple-beacon.sh` by pressing "Ctrl + C".
```

Comment:
Replace first person `I` with either passive voice (the script can be stopped by ....) or second person (you can then stop the script ....)

Reply:
Thanks for the catch!